socket.js

```diff
@@ -7,7 +7,10 @@ var debug = (process.env.APP_DEBUG === 'true' || process.env.APP_DEBUG === true)
 
 var Redis = require('ioredis');
 var redis = new Redis({
-    db: process.env.REDIS_DATBASE || 0
+    port: process.env.REDIS_PORT || 6379, 
+    host: process.env.REDIS_HOST || '127.0.0.1', 
+    db: process.env.REDIS_DATBASE || 0,
```

Comment:
Just noticed, this should be `REDIS_DATABASE` (I know, it was wrong before also). Would you mind changing it?

Thanks